lib/input.rb

```diff
@@ -0,0 +1,44 @@
+require_relative '../lib/board.rb'
+require_relative '../lib/names.rb'
+require_relative '../lib/welcome.rb'
+
+class Input
+  def ask_for_move(players)
+    br_arr = %w[1 2 3 4 5 6 7 8 9]
+    9.times do |x|
```

Comment:
Use a loop that terminates when there is a win or a draw instead of a fixed number of times.